src/inspector_io.cc

```diff
@@ -182,8 +182,11 @@ bool InspectorIo::Start() {
 }
 
 void InspectorIo::Stop() {
+  if (State::kNew == state_)
```

Comment:
Sure, I'll use use the common style, though writing the enum first protects from accidental assignment, it does.